src/test/java/studyconnect/StudyConnectApplicationTest.java

```diff
@@ -0,0 +1,17 @@
+package studyconnect;
+
+import com.studyconnect.StudyConnectApplication;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@SpringBootTest(classes = StudyConnectApplication.class)
+class StudyConnectApplicationTest {
+    @Test
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def modifier' has incorrect indentation level 4, expected level should be 2.